trigger.sh

```diff
@@ -27,6 +27,13 @@ for BRANCH in master R5.0; do
     cd $mydir
     if [ "$result" -ne 0 ]; then
        echo "$(date) Found build number for ${BRANCH}: ${BUILD_NUMBER}, missing in the registry. Starting build." | tee -a "$log_file"
-       zuul -c zuul.conf enqueue-ref --tenant "${TRIGGER_TENANT}" --trigger timer --pipeline "${TRIGGER_PIPELINE}" --project "${TRIGGER_PROJECT}" --ref refs/heads/$BRANCH --newrev "${TRIGGER_NEWREV}"
+       searchStr="${BRANCH}: ${BUILD_NUMBER}, missing in the registry. Starting build"
+       echo $searchStr
+       count=`grep -i "$searchStr" /tmp/zuul_build_trigger.log | wc -l`
```

Comment:
Any reason not to use "$log_file" instead of "/tmp/zuul_build_trigger.log", as in the rest of the script?